presto-spark-base/src/test/java/com/facebook/presto/spark/PrestoSparkQueryRunner.java

```diff
@@ -405,9 +405,9 @@ public PrestoSparkQueryRunner(
     private static void setupLogging()
     {
         Logging logging = Logging.initialize();
-        logging.setLevel("org.apache.spark", INFO);
+        logging.setLevel("org.apache.spark", WARN);
```

Comment:
Can you explain this change? Seems unrelated. If possible, it would be great to extract unrelated changes into a separate PR.

Reply:
Will remove this. Its to prevent flooding logs in CI.